src/neo4j_genai/retrievers/base.py

```diff
@@ -185,11 +190,16 @@ class ExternalRetriever(Retriever, ABC):
     VERIFY_NEO4J_VERSION = False
 
     def __init__(
-        self, driver: neo4j.Driver, id_property_external: str, id_property_neo4j: str
+        self,
+        driver: neo4j.Driver,
+        id_property_external: str,
+        id_property_neo4j: str,
+        neo4j_database: Optional[str] = None,
     ):
         super().__init__(driver)
```

Comment:
The database can be passed to the parent class since ExternalRetriever inherits from Retriever.

Reply:
If just a matter of naming, I'll go for `neo4j_database` everywhere, wdyt?